docs/src/examples/components/Chat/Types/ChatExample.shorthand.tsx

```diff
@@ -6,56 +6,41 @@ const janeAvatar = {
   status: { color: 'green', icon: 'check' },
 }
 
+const content = (
+  <div>
+    Sure! Let's try one of these places:<br />
+    <a href="#">www.goodFood1.com</a>,<br />
+    <a href="#">www.goodFood2.com</a> or<br />
+    <a href="#">www.goodFood3.com</a>
+  </div>
+)
+
 const items = [
-  {
-    content: (
-      <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />
-    ),
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Hi"
-        author="Jane Doe"
-        timestamp="Yesterday, 10:15 PM"
-        avatar={janeAvatar}
-      />
-    ),
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Would you like to grab a lunch?"
-        author="John Doe"
-        timestamp="Yesterday, 10:16 PM"
-        mine
-      />
-    ),
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Sure! Let's try the new place downtown"
-        author="Jane Doe"
-        timestamp="Yesterday, 10:15 PM"
-        avatar={janeAvatar}
-      />
-    ),
-  },
-  {
-    content: <Divider content="Today" type="primary" important />,
-  },
-  {
-    content: (
-      <Chat.Message
-        content="Let's have a call"
-        author="John Doe"
-        timestamp="Today, 11:15 PM"
-        mine
-      />
-    ),
-  },
-]
+  <Chat.Message content="Hello" author="John Doe" timestamp="Yesterday, 10:15 PM" mine />,
```

Comment:
Any particular reason why are we changing the existing examples? It would be hard to notice if there are any styling regression in screener this way. I know you already found one, but it would be much easier to catch all of theme on screener. You can add additional examples, if you want to advertise some different usage.

Reply:
reverted, thanks